libmamba/tests/data/repodata_json_cache/test_4.json

```diff
@@ -1,4 +1,4 @@
-{"_cache_control":"{{}}\",,,\"","_etag":"\n\n\"\"randome ecx,,ssd\n,,\"","_mod":"Fri, 11 Feb 2022 13:52:44 GMT","_url":"file:///Users/wolfvollprecht/Programs/mamba/mamba/tests/channel_a/linux-64/repodata.json",
+{"_cache_control":"{{}}\",,,\"","_etag":"\n\n\"\"random ecx,,ssd\n,,\"","_mod":"Fri, 11 Feb 2022 13:52:44 GMT","_url":"file:///Users/wolfvollprecht/Programs/mamba/mamba/tests/channel_a/linux-64/repodata.json",
```

Comment:
I have no idea what this `_etag` is, but in cpp file is is fixed by typos pre-commit hook, so I changed it here manually.